### DISCUSSION TITLE
tutorials/intro-strawberry/10-resolvers

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#130] [course feedback]
# tutorials/intro-strawberry/10-resolvers

Replace mock data with live data using data sources in your resolvers.

https://www.apollographql.com/tutorials/intro-strawberry/10-resolvers

**Comment by akristen** [comment]
tbh localhost isnt working with the instructions provided. don't know what's causing the error

**Reply by akristen** [reply]
I resolved this + added what I had to do in the REST as data source module!

**Reply by mabuyo** [reply]
Thanks for leaving your note to help others!

**Comment by bparanj** [comment]
Is this happening while the user is interacting in the frontend?

**Reply by lizhennessy** [reply]
Hi there @bparanj - if I'm understanding your question correctly, then yes: if a user was loading a page on the frontend, and that page ran the `FeaturedPlaylists` query that we tested out here in Sandbox, then the response to the query would be the real, live data (and no longer the hardcoded playlist objects). Is that what you meant?
